test/Microsoft.AspNet.Razor.Test/Parser/Html/HtmlDocumentTest.cs

```diff
@@ -70,7 +70,11 @@ public void ParseDocumentCorrectlyHandlesOddlySpacedHTMLElements()
             ParseDocumentTest("<div ><p class = 'bar'> Foo </p></div >",
                 new MarkupBlock(
                     BlockFactory.MarkupTagBlock("<div >"),
-                    BlockFactory.MarkupTagBlock("<p class = 'bar'>"),
+                    new MarkupTagBlock(
+                        Factory.Markup("<p"),
+                        new MarkupBlock(
+                            Factory.Markup(" class")),
+                        Factory.Markup(" = 'bar'>")),
```

Comment:
W.r.t. https://github.com/aspnet/Razor/pull/372/files#r30054041 comment, does seem like handling of whitespace between attribute name and the equals sign is now worse.  How will the `Span` containing `" = 'bar'>"` be treated during code generation e.g. is it ignored?